recipes/comboboxes/combobox_multi_select/combobox_multi_select.vue

```diff
@@ -361,23 +368,36 @@ export default {
     async label () {
       await this.$nextTick();
       // Adjust the chips position if label changed
-      this.setChipsTopPosition();
+      this.setChipsPosition();
     },
 
     async description () {
       await this.$nextTick();
       // Adjust the chips position if description changed
-      this.setChipsTopPosition();
+      this.setChipsPosition();
```

Comment:
Really like the math'ing update you got here. Very readable, and I can see exactly what's going on.